mod/messageboard/start.php

```diff
@@ -114,7 +114,11 @@ function messageboard_add($poster, $owner, $message, $access_id = ACCESS_PUBLIC)
 			$poster->getURL()
 		), $owner->language);
 
-		notify_user($owner->guid, $poster->guid, $subject, $body);
+		$params = [
+			'action' => 'create',
```

Comment:
@juho-jaakkola What would be the downside of distinct action strings? I don't really see any.